### DISCUSSION TITLE
update 169 - Bitdefender spazez out

### DISCUSSION THREAD

**Discussion by brutlern** [Rem0o/FanControl.Releases#2021] [Q&A]
I just installed update 169, and Bitdefender started going nuts, but apparently it's only the update procedure which it doesn't like. Once I restored the quarantined files, update 169 was up and running. Scanning all the new files does not trigger Bitdefender, but as soon as it tries to update (I restored 168 to verify this), Bitdefender doesn't like it at all. 
Now I know what you're gonna say, "False positive", but the fact is, Bitdefender was fine with any of the previous updates, but something changed with 169, which triggers Bitdefender.

**Comment by Rem0o** [comment]
My guess is the SYSTEM task, which made it to the version by mistake. Will revert that asap.

**Comment by brutlern** [comment]
Will see if it's fixed in 170.
